Fix typos (#2088)
gohugoio/hugoDocs: content/en/content-management/multilingual.md

{{< new-in "0.112.0" >}}

> In version `0.112.0` of Hugo we consolidated all configuration options, but also improved how the languages and their params get merged with the main configuration. But while testing this on Hugo sites out there, we got some error reports and reverted some of the changes in favour of deprecation warnings:

Reviewer: ```suggestion
In Hugo `v0.112.0` we consolidated all configuration options, and improved how the languages and their parameters are merged with the main configuration. But while testing this on Hugo sites out there, we received some error reports and reverted some of the changes in favor of deprecation warnings:
```

color = "blue"
```

> In the example above, all the settings exept the `color` below `params` maps to predefined configuration options in Hugo for the site and its language and should be accessed via the documented accessors:

Reviewer: ```suggestion
In the example above, all settings except `color` below `params` map to predefined configuration options in Hugo for the site and its language, and should be accessed via the documented accessors:
```
